x-pack/plugins/enterprise_search/public/applications/workplace_search/utils/mime_types.ts

```diff
@@ -51,4 +53,5 @@ const mimeTypes = {
   'video/quicktime': 'MOV',
 } as { [key: string]: string };
 
-export const mimeType = (type: string) => mimeTypes[type.toLowerCase()] || type;
+export const mimeType = (type: CustomAPIFieldValue) =>
+  mimeTypes[type.toString().toLowerCase()] || type;
```

Comment:
I didn't update tests here, because the case "Returns `type` if it's not found in `mimeTypes`" is already covered.